tools/travis/build.sh

```diff
@@ -24,7 +24,7 @@ cd $ROOTDIR
 
 cd $ROOTDIR/ansible
 
-$ANSIBLE_CMD openwhisk.yml
+$ANSIBLE_CMD openwhisk_fresh.yml
```

Comment:
@domdom82 @lehoanganh @markusthoemmes this is the only play to change the usage of "openwhisk.yml" ?